ssas/connection.go

```diff
@@ -1,35 +1,53 @@
 package ssas
 
 import (
+	"database/sql"
 	"os"
-	"runtime"
+	"time"
 
 	_ "github.com/lib/pq"
-	log "github.com/sirupsen/logrus"
 	"gorm.io/driver/postgres"
 	"gorm.io/gorm"
 )
 
-// Variable substitution to support testing.
-var LogFatal = log.Fatal
+var Connection *gorm.DB
+
+func init() {
+	var err error
+	Connection, err = createDB()
 
-func GetGORMDbConnection() *gorm.DB {
-	databaseURL := os.Getenv("DATABASE_URL")
-	db, err := gorm.Open(postgres.Open(databaseURL), &gorm.Config{})
 	if err != nil {
-		LogFatal(err)
+		Logger.Fatalf("Failed to create db %s", err.Error())
 	}
-	return db
 }
 
-func Close(db *gorm.DB) {
```

Comment:
We don't need this method anymore because we never want to forcefully close the gorm DB pool. `Connection` will self-manage the creation and removal of database connections based on the configuration (MaxOpenConns, MaxIdleConns, ConnMaxLifetime)